<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QWidget" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>381</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Add Mode</string>
    </attribute>
    <widget class="QPushButton" name="bTextureListRefresh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bToggleGridLines">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>290</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle Grid Lines</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Tiles</string>
     </property>
     <widget class="QListWidget" name="lwTextureList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bTest">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Tile</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbAddTileCounter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Enemies</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Enemy</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lwEnemyList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="bClearTileBuffer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Buffer</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Exits</string>
     </property>
     <widget class="QListWidget" name="lwExitList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bAddExit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Coins</string>
     </property>
     <widget class="QListWidget" name="lwCoinList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbAddCoinCounter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="bAddCoin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Coin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Score:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbCoinScoreValue">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Selection Mode</string>
    </attribute>
    <widget class="QPushButton" name="bDisableSelectionMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Selection Mode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bEnableSelectionMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable Selection Mode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbOutline">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Outline</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>211</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Entity Properties</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Position X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Position Y:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbRealTime">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Real-time</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="bApplyPosition">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Position</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbSelectedEntityPositionX">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
      <property name="singleStep">
       <number>50</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbSelectedEntityPositionY">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
      <property name="singleStep">
       <number>50</number>
      </property>
     </widget>
     <widget class="QPushButton" name="bDeleteSelectedEntity">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Entity</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Map Width:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="MapWidth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>800</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="bApplyMapWidth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Import/Export</string>
    </attribute>
    <widget class="QLineEdit" name="tbExportFileName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="bExport">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>EXPORT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bImport">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>IMPORT</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbImportFileName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="bRefreshImportFileList">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time Limit:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbTimeLimit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="bCleanMap">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bRestartMap">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>RESTART</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbPlayerName">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player Name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
